R/pkgs/hospitalization/DESCRIPTION

```diff
@@ -0,0 +1,24 @@
+Package: hospitalization
+Title: Generate hospitalization scenarios corresponding to the infection scenarios from a run of the SEIR code.
+Version: 0.0.1
+Imports:
+    magrittr,
+    knitr,
+    tidyr,
+    dplyr,
+    readr,
+    gridExtra,
+    ggfortify,
+    flextable,
+    doParallel,
+    data.table,
+    foreach
+Authors@R: 
```

Comment:
If you want to be the author/maintainer that's fine, but if you don't want to you can set me jkaminsky@jhu.edu